stat/distuv/gamma.go

```diff
@@ -135,18 +138,19 @@ func (g Gamma) Rand() float64 {
 			} else {
 				z = -exprnd() / lambda
 			}
-			lh := lc - z - math.Exp(-z/a)
+			eza := math.Exp(-z / a)
+			lh := -z - eza
 			var lEta float64
 			if z >= 0 {
-				lEta = lc - z
+				lEta = -z
 			} else {
-				lEta = lc + lw + math.Log(lambda) + lambda*z
+				lEta = -1 + lambda*z
 			}
 			if lh-lEta > -exprnd() {
-				return math.Exp(-z/a) / b
+				return eza / b
 			}
```

Comment:
Here too.

Reply:
`lc` is unnecessary (we only ever need the difference `lh - lEta`).

Reply:
Sorry, missed this. This is nice.